chart/templates/_helpers.tpl

```diff
@@ -33,3 +33,9 @@ chart: {{ .Chart.Name }}-{{ .Chart.Version }}
 heritage: {{ .Release.Service }}
 release: {{ .Release.Name }}
 {{- end }}
+
+{{- define "rancher.systemDefaultRegistry" -}}
```

Comment:
Either: 
```suggestion
{{- define "system_default_registry" -}}
```
or
```suggestion
{{- define "rancher.system_default_registry" -}}
```
I don't know why we put `rancher.` in there

Reply:
I was trying to follow what the [Helm documentation](https://helm.sh/docs/chart_template_guide/named_templates/) recommends:

> By using the specific chart name as a prefix we can avoid any conflicts that may arise due to two different charts that implement templates of the same name.

But since no conflict will happen here, I changed it to `system_default_registry`